infrastructure/docker/services/php-base/app.ini

```diff
@@ -1,3 +1,4 @@
 date.timezone=Europe/Paris
 upload_max_filesize=20M
 post_max_size=20M
+memory_limit=512M
```

Comment:
This should be applied only for CLI SAPI.
Here it's also applied to PHP-FPM SAPI.
For FPM, the value should stay very small

Reply:
Fixed in 43251ae